repositories.yaml

```diff
@@ -1406,3 +1406,23 @@ applications:
     moz_pipeline_metadata_defaults:
       expiration_policy:
         delete_after_days: 400
+
+  - app_name: moso_mastodon_web
```

Comment:
Will this have backend component too? We seem to be adopting an unwritten convention of `_frontend` and `_backend` suffixes to app_name.

Reply:
@akkomar yes, but MoSo has more than one frontend: we changed the prefix to match the unwritten convention, but having "_frontend_web" seemed to violate such a convention as well

Reply:
@akkomar see https://bugzilla.mozilla.org/show_bug.cgi?id=1864983#c1

Reply:
Ah, sorry, I should have read the bug.